modules/hooks.nix

```
@@ -359,6 +359,13 @@ in
           entry = "${pkgs.python3Packages.black}/bin/black";
           types = [ "file" "python" ];
         };
+      ruff =
+        {
+          name = "ruff";
+          description = " An extremely fast Python linter, written in Rust.";
+          entry = "${pkgs.python3Packages.black}/bin/ruff";
```

```suggestion
          entry = "${pkgs.ruff}/bin/ruff";
```

`ruff` is not a Python package.